tests/unit/jest.config.js

```diff
@@ -13,16 +13,25 @@ const preactModuleMapper =
 
 const conditionalWebComponentsConfig = useWebComponents
   ? {
-      testMatch: ['<rootDir>/design-system/src/components/web-components/**/*.test.[jt]s(x)?'],
+      testMatch: [
+        '<rootDir>/packages/design-system/src/components/web-components/**/*.test.[jt]s(x)?',
+        '<rootDir>/tests/browser/custom-reporter.test.ts',
+      ],
+      testPathIgnorePatterns: ['<rootDir>/tests/browser/'],
     }
-  : { testPathIgnorePatterns: ['<rootDir>/design-system/src/components/web-components'] };
+  : {
+      testPathIgnorePatterns: [
+        '<rootDir>/packages/design-system/src/components/web-components',
+        '<rootDir>/tests/browser/',
```

Comment:
Why is /tests/browser ignored here?

Reply:
Ooh, yes good catch, I think the logic of this configuration could use some clarity. It seems that if I remove <rootDir>/tests/browser/ from the testPathIgnorePatterns list when useWebComponents is set to false, we end up overwriting it in the main configuration spread, [here](https://github.com/CMSgov/design-system/blob/main/tests/unit/jest.config.js#L26-L33). This results in unintentionally including the playwright tests in our jest runs.

Reply:
Ah yeah ok this makes sense I think. Is this happening because of the changes to the rootDir and coverageDirectory changes below?